PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -1,133 +1,146 @@
-		
-		<v-checkbox
-			v-model="checkbox"
-			label="I have no email address"
-		></v-checkbox>
-		</v-col>
-
-<v-row>
-	<v-radio-group
-        v-model="test"
-        label="May we contact you regarding follow up vaccination information?"
-      >
-        <v-col align="right" cols="3" sm="3" md="3">
-          <v-radio label="Yes" value="yes"></v-radio>
-          <v-radio label="No" value="no"></v-radio>
-        </v-col>
-      </v-radio-group>
-	
-</v-row>
-
-</v-container>
+			</v-col>
+		</v-row>
+		<v-row>
+			<v-col cols="6" sm="6" md="3">
+				<v-checkbox
+					v-model="checkbox"
+					label="I have no phone number"
+				></v-checkbox>
+			</v-col>
+		</v-row>
+		<v-row>
+			<v-col cols="12" sm="12" md="6">
+				<v-text-field
+					required
+					:rules="emailRules"
+					label="Primary E-mail Address"
+					v-model="primaryEmail"
+				></v-text-field>
+			</v-col>
+			<v-col cols="12" sm="12" md="6">
+				<v-text-field
+					:rules="emailRules"
+					label="Secondary E-mail Address"
+					v-model="secondaryEmail"
+				></v-text-field>
+			</v-col>
+		</v-row>
+			<v-col cols="6" sm="6" md="3">
+				<v-checkbox
+					v-model="checkbox"
+					label="I have no email address"
+				></v-checkbox>
+			</v-col>
+		<v-row>
+			<v-radio-group
+				required
+				:rules="[v => !!v || 'This field is required']"
+				v-model="approval"
+				label="May we contact you regarding follow up vaccination information?"
+			>
+				<v-col align="right" cols="3" sm="3" md="3">
+					<v-radio label="Yes" value="yes"></v-radio>
+					<v-radio label="No" value="no"></v-radio>
+				</v-col>
+			</v-radio-group>
+		</v-row>
+	</v-container>
 </template>
 
 <script>
+import EventBus from '../eventBus'
 
   export default {
     name: 'SinglePatientContactInfo',
 	data () {
 		return { 
 			phonetype:['Cell', 'Landline', 'Other'],
-			radios: 'May we contact you for a follow up vaccination?'
+			radios: 'May we contact you for a follow up vaccination?',
+			emailRules: [
+				(v) =>
+				/^[\s]*$|.+@.+\..+/.test(v) ||
+				"E-mail must be in the format example@example.com",
+			],
+			primaryPhoneNumber: '',
+			primaryPhoneNumberType: '',
+			primaryEmail: '',
+			secondaryPhoneNumber: '',
+			secondaryPhoneNumberType: '',
+			secondaryEmail: '',
+			approval: ''
 		}
+	},
+	methods: {
+		sendHouseholdContactInfoInfoToReviewPage()
+		{
+			const householdContactInfoPayload = {
+				primaryPhoneNumber: this.primaryPhoneNumber,
+				primaryPhoneNumberType: this.primaryPhoneNumberType,
+				primaryEmail: this.primaryEmail,
+				secondaryPhoneNumber: this.secondaryPhoneNumber,
+				secondaryPhoneNumberType: this.secondaryPhoneNumberType,
+				secondaryEmail: this.secondaryEmail,
+				approval: this.approval
+			}
+			EventBus.$emit('DATA_HOUSEHOLD_CONTACT_INFO_PUBLISHED', householdContactInfoPayload)
+		},
+		verifyFormContents()
+		{
+			var valid = true
+			var message = "Woops! You need to enter the following fields:"
 		
+			if(this.primaryPhoneNumber == "") {
```

Comment:
For all of these fields, I think we should also check if the prop is null as well as empty (you're doing empty rn). I'm pretty sure Vue handles these differently since most javascript-y languages treat null/empty differently. 

That might take the form as `if(!this.primaryPhoneNumber && this.primaryPhoneNumber == "") {}`. I was reading online and this seems like a cleaner implementation - the example explicitly says `this.$options.propData` so I figure `options` or `propData` would be replaced with `primaryPhoneNumber`. 

This also comes with another question - when you make these fields, do you initialize or default the prop to be null? The bottom of this page indicates possibly yes, but I'm not sure if they're using an eventbus https://forum.vuejs.org/t/is-there-any-way-to-check-if-all-props-passed-the-validation/26547/2

Reply:
This wasn't part of my change - if GitHub says it is, perhaps it was due to a manual merge or rearrangement of lines.  Create a separate card for this?

Reply:
Yea, it's probably because of how you wrote some code and shifted something down. A separate card would work!

Reply:
Created MVS-396